File: app/src/main/java/com/example/weatherchallengemvvmkotlin/data/models/SysObject.java
package com.example.weatherchallengemvvmkotlin.data.models;

import com.google.gson.annotations.SerializedName;

public class SysObject {

    @SerializedName("message")
    double message;

    @SerializedName("country")
    String country;

    @SerializedName("sunrise")
    long sunrise;

    @SerializedName("sunset")
    long sunset;

    public double getMessage() {
        return message;
    }

    public String getCountry() {
        return country;
    }

    public long getSunrise() {
        return sunrise;
    }

    public long getSunset() {
        return sunset;
    }
}

File: app/src/main/java/com/example/weatherchallengemvvmkotlin/data/models/SysObject.kt
package com.example.weatherchallengemvvmkotlin.data.models

import com.google.gson.annotations.SerializedName

class SysObject {
    @SerializedName("message") var message : Double = 0.0;
    @SerializedName("country") var country : String? = null
    @SerializedName("sunrise") var sunrise : Long = 0
    @SerializedName("sunset") var sunset : Long = 0
}